WinUIGallery/Controls/DesignGuidance/ColorTile.xaml

```diff
@@ -53,7 +53,225 @@
+                                                        KeySpline="0.39,0.0 0.63,1.0"
+                                                        KeyTime="0:0:0.267"
+                                                        Value="1.146" />
+                                                    <SplineDoubleKeyFrame
+                                                        KeySpline="0.55,0.0 0.02,1.0"
+                                                        KeyTime="0:0:0.333"
+                                                        Value="1.0" />
+                                                </DoubleAnimationUsingKeyFrames>
+                                            </Storyboard>
+                                        </Grid.Resources>
+
+                                        <Grid.BackgroundTransition>
+                                            <BrushTransition Duration="0:0:0.083" />
+                                        </Grid.BackgroundTransition>
+                                        <VisualStateManager.VisualStateGroups>
+                                            <VisualStateGroup x:Name="CommonStates">
+                                                <VisualState x:Name="Normal" />
+                                                <VisualState x:Name="PointerOver">
+                                                    <Storyboard>
+                                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="RootGrid" Storyboard.TargetProperty="Background">
+                                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonBackgroundPointerOver}" />
+                                                        </ObjectAnimationUsingKeyFrames>
+                                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="RootGrid" Storyboard.TargetProperty="BorderBrush">
+                                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonBorderBrushPointerOver}" />
+                                                        </ObjectAnimationUsingKeyFrames>
+                                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="Foreground">
+                                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{Binding Foreground, RelativeSource={RelativeSource Mode=TemplatedParent}}" />
+                                                        </ObjectAnimationUsingKeyFrames>
+                                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="CopySuccessGlyph" Storyboard.TargetProperty="Foreground">
+                                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{Binding Foreground, RelativeSource={RelativeSource Mode=TemplatedParent}}" />
```

Comment:
You should be able to directly update the copyButton controls under .\controls\CopyButton\CopyButton.xaml

This will avoid the redundancy to re-template here.

Reply:
The `ButtonForegroundPointerOver` and `ButtonBackgroundPressed` is not purely black and purely white in the original template, and you have set `Foreground` in the original template so `TargetNullValue` cannot be used.

Reply:
I agree this isn't the solution.